docs/supported-services/s3.rst

```diff
@@ -54,12 +53,80 @@ This service takes the following parameters:
      - No
      - disabled
      - Whether to enable logging on the bucket. Allowed values: *enabled*, *disabled*.
+   * - lifecycles
+     - :ref:`s3-lifecycles`
+     - No
+     - disabled
+     - Lifecycle Policies to apply to the bucket. See `AWS Docs for more info <http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-lifecycleconfig-rule.html>`_
    * - tags
      - :ref:`s3-tags`
      - No
      -
      - Any tags you want to apply to your S3 bucket
 
+.. _s3-lifecycles:
+
+lifecycles
+~~~~~~~~~~
+A list of life cycle rules
```

Comment:
It looks like you're missing docs for the "prefix" field. Is that on the "lifecycles" section or under each "transitions" section?